### DISCUSSION TITLE
sms automation

### DISCUSSION THREAD

**Discussion by radu-tataru** [Budibase/budibase#5239] [Help]
Hi. I really like budibase and i am trying to build a small application with it, as a proof of concept for a future client (an auto repair shop). I am now struggling to automate an SMS notification system. These SMSs are reminders for the shop apointments.
My approach is to set a cron trigger that runs every morning at 8 AM, reads all the rows from the appointments table that have the appintment date = {{ date now }} and for each row i generate a webhook for zapier (SMS.to) with the phone number, client name etc as payload.
My question is: how do i create a webhook for each row (is this possible or my whole approach is wrong?). If my approach is wrong how should i try to accomplish this?
Thank you!

**Comment by melohagan** [comment]
Hi @rad0slav 

Functionality for looping over rows in automation is raised in this issue here: https://github.com/Budibase/budibase/issues/4427
